default/stringtables/en.txt

```diff
@@ -7215,7 +7225,9 @@ SITREP_SHOWALL_TEMPLATES
 Show all blocked SitReps
 
 SITREP_WELCOME
-[[encyclopedia GREETINGS_GUIDE_TITLE]] [[GREETINGS_GUIDE_REFERENCE]]
+'''[[encyclopedia GREETINGS_GUIDE_TITLE]] [[GREETINGS_INTRO]]
+If you are a newcomer to this universe, please click on the word "[[GREETINGS_GUIDE_TITLE]]" to help us get you started.
```

Comment:
I'd rephrase to: "... for some basic advice on how to play." (developers don't need mentioning)

Reply:
the rest of the text I didn't make suggestions for also needs similar rewriting

Reply:
oh kay. i'll have a fresh look at it next week.